part2/stages/TPM-Check

```diff
@@ -167,25 +167,6 @@ case $? in
         ;;
 esac
 
-# determine if TPM owner password is WKS
-# this isn't really necessary yet since we error out if TPM is owned
-tpm_is_owner_wks
-case $? in
-    0)  TSS_WELL_KNOWN_SECRET='true' ;;
-    1)  # this is an error condition but since we refuse to continue
-        # if TPM is owned we won't get here ... yet
-        TSS_WELL_KNOWN_SECRET='false' ;;
```

Comment:
With `TSS_WELL_KNOWN_SECRET` no longer set, should the now empty variables still be appended in [MEASURED_LAUNCH_CONF]( https://github.com/jandryuk/installer/blob/cba9d033ab577a635eedc4c1e82120c62a03c82b/part2/stages/TPM-Check#L228) later? I do not find a script relying on it, but I might be missing something.

Reply:
Yes, you are right.  Good catch.  I'll re-spin with the removal.